radio/util/font2png.py

```diff
@@ -1,119 +1,96 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import sys
-try:
-    from PyQt5 import Qt, QtGui, QtCore
-except:
-    from PyQt4 import Qt, QtGui, QtCore
-
+import sys, os
```

Comment:
2 different lines (PEP8 if I am not mistaken)

Reply:
forget it, the whole file was not PEP8 compliant anyhow (at least function names should use _ in names)